api/v1alpha1/podiochaos_types.go

```diff
@@ -124,6 +131,32 @@ type AttrOverrideSpec struct {
 	Rdev *uint32 `json:"rdev,omitempty"`
 }
 
+// MistakeSpec represents one type of mistake
+type MistakeSpec struct {
+	// Filling determines what is filled in the miskate data.
+	//+optional
```

Comment:
```suggestion
	// +optional
```

Appending one whitespace at the beginning of the comment. 

Although it does not affect the code generation, we should keep the same style.

Reply:
Similar situations in other fields.